doc/flame/components.md

```diff
@@ -1104,6 +1104,21 @@ Check the example app
 for details on how to use it.
 
 
+## LayoutComponent
```

Comment:
If we won't expose this we should change the docs to be about `RowComponent` and `ColumnComponent` instead.